src/cloudai/metrics.py

```
@@ -0,0 +1,264 @@
+            raise ValueError(f"Unknown SOL metric '{key}'. Available metrics: {available}") from exc
+
+    @classmethod
+    def get_dimension(cls, key: str) -> DimensionDefinition:
+        try:
+            return cls._dimensions[key]
+        except KeyError as exc:
+            available = ", ".join(sorted(cls._dimensions))
+            raise ValueError(f"Unknown metric dimension '{key}'. Available dimensions: {available}") from exc
+
+    @classmethod
+    def register_metrics(cls, *metrics: MetricDefinition):
```

Ruff (0.16.1)

[warning] Missing return type annotation for classmethod `register_metrics`

Add return type annotation: `None`

(ANN206)